specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/file.json

```diff
@@ -0,0 +1,1094 @@
+            "modelAsString": true
+          },
+          "description": "Specifies whether the lease on a share is of infinite or fixed duration, only when the share is leased."
+        },
+        "signedIdentifiers": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SignedIdentifier"
+          },
+          "description": "List of stored access policies specified on the share."
+        },
+        "snapshotTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "Creation time of share snapshot returned in the response of list shares with expand param \"snapshots\"."
+        }
+      },
+      "description": "The properties of the file share."
+    },
+    "LeaseShareRequest": {
+      "properties": {
+        "action": {
+          "type": "string",
+          "enum": [
+            "Acquire",
+            "Renew",
+            "Change",
+            "Release",
+            "Break"
+          ],
+          "description": "Specifies the lease action. Can be one of the available actions."
+        },
+        "leaseId": {
+          "type": "string",
+          "description": "Identifies the lease. Can be specified in any valid GUID string format."
+        },
+        "breakPeriod": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Optional. For a break action, proposed duration the lease should continue before it is broken, in seconds, between 0 and 60."
+        },
+        "leaseDuration": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Required for acquire. Specifies the duration of the lease, in seconds, or negative one (-1) for a lease that never expires."
+        },
+        "proposedLeaseId": {
+          "type": "string",
+          "description": "Optional for acquire, required for change. Proposed lease ID, in a GUID string format."
+        }
+      },
+      "required": [
+        "action"
+      ],
+      "description": "Lease Share request schema."
+    },
+    "LeaseShareResponse": {
+      "properties": {
+        "leaseId": {
+          "type": "string",
+          "description": "Returned unique lease ID that must be included with any request to delete the share, or to renew, change, or release the lease."
+        },
+        "leaseTimeSeconds": {
+          "type": "string",
+          "description": "Approximate time remaining in the lease period, in seconds."
+        }
+      },
+      "description": "Lease Share response schema."
+    },
+    "SignedIdentifier": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "An unique identifier of the stored access policy."
+        },
+        "accessPolicy": {
+          "$ref": "#/definitions/AccessPolicy",
+          "description": "Access policy"
+        }
+      }
+    },
+    "AccessPolicy": {
+      "properties": {
+        "start": {
```

Comment:
>start [](http://example.com/codeflow?start=9&length=5)

Should use format: date-time (same for expiry)

Reply:
will fix